src/OVAL/public/oval_definitions.h

```diff
@@ -1828,15 +1901,15 @@ struct oval_component *oval_variable_get_component(struct oval_variable *);	//ty
  * It should be freed after use by the calling application.
  * @memberof oval_variable
  */
-struct oval_iterator *oval_variable_get_possible_values(struct oval_variable *variable);
+struct oval_variable_possible_value_iterator *oval_variable_get_possible_values(struct oval_variable *variable);
```

Comment:
Breaks it how?  As far as I could tell it was already broken, at least from the python perspective.  I had access to oval_variable_get_possible_values() which returned an oval_iterator but there are zero exposed functions that can work with a plain oval_iterator.  I considered exposing the plain functions, but decided it would be better to make special iterator functions like you do for every other iterator.